Mage/src/main/java/mage/abilities/condition/common/PlayerDiscardedThisTurnCondition.java

```diff
@@ -0,0 +1,24 @@
+package mage.abilities.condition.common;
+
+import mage.abilities.Ability;
+import mage.abilities.condition.Condition;
+import mage.game.Game;
+import mage.watchers.common.DiscardedCardWatcher;
+
+/**
+ *
+ * @author weirddan455
+ */
+public enum PlayerDiscardedThisTurnCondition implements Condition {
```

Comment:
I haven't been looking at spoilers, but did you make this a common condition (rather than one inside the Raven Man) because it's used in several of the spoiled cards already?

Reply:
Not used anywhere else yet.  I just figured it might as be common to take advantage of the method I created in the existing common Watcher.  It could be private if you prefer.